plugins/eksporter.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: eksporter
+spec:
+  version: "v1.0.0"
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/Kyrremann/kubeflow-eksporter/releases/download/v1.0.0/eksporter.tar.gz
+    sha256: "1c13fac59c1c306db8498f394ee4cf1ba98eaa44c6f404fc2fad505533f00835"
+    files:
+    - from: "/eksporter.rb"
+      to: "."
+    bin: "./eksporter.rb"
+  shortDescription: Eksport resources without generated fields
+  homepage: https://github.com/Kyrremann/kubeflow-eksporter
+  caveats: |
+    This plugin needs the following programs:
+    * ruby
```

Comment:
Is the `yaml` library built-in in ruby, or does the user have to install some gem for this?

Reply:
Yaml is built-in. At least from `Ruby-1.9.1`.